docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1023,6 +1105,26 @@ The following table lists all the search category attributes the server can rece
 >
 > These Attributes support Tizen 5.0 and Higher for Mobile and Wearable.
 
+## Media Controller Search Category Attributes
```

Comment:
Would you please check this title? Is this table really related with Search?

Reply:
Done